src/common/options.cc

```diff
@@ -1881,71 +1881,315 @@ std::vector<Option> global_options = {
 
   Option("osd_op_queue", Option::TYPE_STR, Option::LEVEL_ADVANCED)
   .set_default("wpq")
-  .set_description(""),
+  .set_description("which operation queue algorithm to use; can be one of five values -- 'wpq', 'prioritized', 'mclock_opclass', 'mclock_client', and 'debug_random'")
+  .add_service("osd")
+  .add_see_also("osd_op_queue_cut_off"),
 
   Option("osd_op_queue_cut_off", Option::TYPE_STR, Option::LEVEL_ADVANCED)
   .set_default("low")
-  .set_description(""),
+  .set_description("the threshold between high priority ops that use strict priority ordering and low priority ops that use a fairness algorithm that may or may not incorporate prioirty; can be one of three values -- 'low', 'high', and 'debug_random'")
```

Comment:
So I did that. I looked through options.h to see if there was any guidance about how long a description (not long description) would ideally be and couldn't find anything. Any thoughts?